hashed_partitioning/README.md

```diff
@@ -0,0 +1,104 @@
+# Hashed Partitioning with DuckDB
+
+This cookbook demonstrates how to use the Spice OSS runtime to accelerate a dataset into multiple partitioned DuckDB files.
```

Comment:
Explain further what hashed partitioning is, why you want to do it, and that this cookbook shows you how accelerate & query.